.github/workflows/pull-request.yaml

```diff
@@ -49,17 +49,6 @@ jobs:
     - run: go get -u github.com/google/addlicense
     - run: addlicense -check .
   
-  npm-audit-advisory:
```

Comment:
I think it's valuable to still have `npm audit` in PR, as long as it's not blocking.

Reply:
This hasn't been removed, just moved to another file